pkg/machine/e2e/scripts/fake_compose.bat

```diff
@@ -0,0 +1,5 @@
+@ECHO off
+set psexe=powershell
+where pwsh > nul 2> nul
+if %errorlevel% == 0 (set psexe="pwsh")
+%psexe% -ExecutionPolicy Unrestricted -File %~dp0\fake_compose.ps1 %*

```

Comment:
instead of needing two files can you not just use
`echo %DOCKER_HOST%`, etc.. here instead of having to call another file.